Objects/TI4_Helpers/TI4_TechnologyHelper.ttslua

```diff
@@ -0,0 +1,174 @@
+function getHelperClient(helperObjectName)
+    local function getHelperObject()
+        for _, object in ipairs(getAllObjects()) do
+            if object.getName() == helperObjectName then return object end
+        end
+        error('missing object "' .. helperObjectName .. '"')
+    end
+    local helperObject = false
+    local function getCallWrapper(functionName)
+        helperObject = helperObject or getHelperObject()
+        if not helperObject.getVar(functionName) then error('missing ' .. helperObjectName .. '.' .. functionName) end
+        return function(parameters) return helperObject.call(functionName, parameters) end
+    end
+    return setmetatable({}, { __index = function(t, k) return getCallWrapper(k) end })
+end
+
+local _technologies = {
+	["Neural Motivator"] = { type = "Green", requirements = nil, abbrev = 'Neural', expansion="base" },
```

Comment:
Other helpers use `source="PoK"` rather an `expansion`, switch to source for consistency?

Reply:
I'll just change it then. Should I keep source="base" for base-game components or just remove the entry to let calls return nil?

Reply:
faction helper doesn't specify a source for the base factions, but deck helper uses 'Base'.  Game setup is the only consumer right now, and it only looks for "PoK" or "Codex 1" so at the moment it doesn't matter.